### PR TITLE
Remove suspend grace period

### PR DESCRIPTION
**Pull request by jekhor**
Change membership fees accounting logic:
- Suspend user immediately when membership fee expired
- Notify user about expiring of the fee starting from 7 days before  expiration.
- Notify members Telegram chat about expiring of the user's membership fee starting from 3 days before expiration
- When user was suspended, allow to unsuspend only if he paid fo 14 days or more.

TODO:
Test if working :)
Cover this changes by tests.
Make code more clean

Closes: #537

### CODE REVIEW

File: app/models/payment.rb [hunk 1]
```diff
@@ -58,9 +58,10 @@ def start_date_before_of_end_date
 
   def set_user_as_unsuspended
     if user.present?
+      return unless user.account_suspended?
       return if user.account_banned?
 
-      if user.last_payment && (user.last_payment.end_date >= Date.today ) && user.account_suspended?
+      if user.last_payment && (user.last_payment.end_date - user.first_payment_after_last_suspend.start_date + 1.day >= 14.days)
```
Comment (yahor) [inline app/models/payment.rb:64]: user.first_payment_after_last_suspend можа быць nil ?

Comment (jekhor) [reply app/models/payment.rb:64]: Тэарэтычна можа, але гэты метад вызываецца пры дадаванні плацяжа, калі юзэр у саспендзе. Таму не павінны. Але праверку трэба.

File: app/models/user.rb [hunk 2]
```diff
@@ -253,8 +253,7 @@ def suspend!
 
     transaction do
       #simple update without callbacks
-      update_column(:account_suspended, true)
-      update_column(:suspended_changed_at, Time.now)
+      update_columns(account_suspended: true, suspended_changed_at: Time.now)
```
Comment (jekhor) [inline app/models/user.rb:256]: О, а так можна было?.. :)

Comment (abitrolly) [reply app/models/user.rb:256]: В клюбе любителей суспензии только так и можно.

**Review by yahor — APPROVED** [review]

**Pull request merged**